Document missing Allocation Service helm variables (#3053)
googleforgames/agones: site/content/en/docs/Installation/Install Agones/helm.md

| `agones.allocator.replicas`                              | The number of replicas to run in the deployment                                                                                                                                                                         | `3`                                |
| `agones.allocator.service.name`                          | Service name for the allocator                                                                                                                                                                                          | `agones-allocator`                 |
| `agones.allocator.service.serviceType`                   | The [Service Type][service] of the HTTP Service                                                                                                                                                                         | `LoadBalancer`                     |

> | `agones.allocator.service.http.nodePort`, `agones.allocator.service.grpc.nodePort`                   | The [NodePort] serves as the external entry point for incoming requests for your app                                                                                                                                                                         | `30000-32767`                     |

Reviewer: Let's not combine the two, but have separate rows for each please.

We should also make clear - This parameter is only used if the ServiceType is set to "NodePort".

Thanks!

Author: These two parameters are added in separate rows. Thanks!

| `agones.allocator.replicas`                              | The number of replicas to run in the deployment                                                                                                                                                                         | `3`                                |
| `agones.allocator.service.name`                          | Service name for the allocator                                                                                                                                                                                          | `agones-allocator`                 |
| `agones.allocator.service.serviceType`                   | The [Service Type][service] of the HTTP Service                                                                                                                                                                         | `LoadBalancer`                     |

> | `agones.allocator.service.http.nodePort`                   | The NodePort parameter is used if the ServiceType is set to "NodePort" and it serves as the external entry point for the incoming requests                                                                                                                                                                         | `30000-32767`                     |

Reviewer: ```suggestion
| `agones.allocator.service.http.nodePort`                   | If the ServiceType is set to "NodePort",  this is the NodePort that the allocator http service is exposed on.                                                                                                                                                                      | `30000-32767`                     |
```

Author: Modified it.

| `agones.allocator.service.http.targetPort`               | The port that is used by the allocator pod to listen for [REST requests][rest-requests]. Note that the allocator server cannot bind to low numbered ports.                                                              | `8443`                             |
| `agones.allocator.service.grpc.enabled`                  | If true the [allocator service][allocator] will respond to [gRPC requests][grpc-requests]                                                                                                                               | `true`                             |
| `agones.allocator.service.grpc.port`                     | The port that is exposed externally by the [allocator service][allocator] for [gRPC requests][grpc-requests]                                                                                                            | `443`                              |
| `agones.allocator.service.grpc.portName`                     | The name of exposed port                                                                                                            | ``                              |

> | `agones.allocator.service.grpc.nodePort`                   | The NodePort parameter is used if the ServiceType is set to "NodePort" and it serves as the external entry point for the incoming requests                                                                                                                                                                          | `30000-32767`                     |

Reviewer: ```suggestion
| `agones.allocator.service.grpc.nodePort`                   | If the ServiceType is set to "NodePort",  this is the NodePort that the allocator gRPC service is exposed on.                                                                                                                                                                  | `30000-32767`                     |
```
